src/services/CreateAnalysisService.tsx

```diff
@@ -21,22 +21,28 @@ class CreateAnalysisService {
 
   static formatDate(dateStr: string): string {
```

Comment:
If time permits, I would suggest moving this function to a utils file for reusability.

Reply:
Great idea, I moved that function alone with 2 other commonly used utilities into their separate file